src/main/scala/scorex/api/http/AddressApiRoute.scala

```diff
@@ -228,6 +238,19 @@ case class AddressApiRoute(settings: RestAPISettings, wallet: Wallet, state: Sta
       .getOrElse(InvalidAddress)
   }
 
+  private def balancesDetailsJson(address: String)(implicit transactionModule: TransactionModule): ToResponseMarshallable = {
```

Comment:
The idea of this class is to provide routing, business logic is best to be kept elsewhere